test/config-next/ca.json

```diff
@@ -65,9 +65,9 @@
 			"issuers": [
 				{
 					"active": true,
-					"issuerURL": "http://127.0.0.1:4502/int ecdsa a",
-					"ocspURL": "http://127.0.0.1:4002/",
-					"crlURLBase": "http://127.0.0.1:4501/ecdsa-a/",
+					"issuerURL": "http://boulder.ca:4502/int-ecdsa-a",
+					"ocspURL": "http://boulder.ca:4002/",
+					"crlURLBase": "http://boulder.ca:4501/ecdsa-a/",
```

Comment:
It's a minor nit since it'll never really be resolved externally, but should we use a domain we own?  We have a wide variety to choose from

Reply:
On the one hand, I kind of specifically don't want to use a "real" domain name, lest we accidentally come to rely on this test-only URL being something specific.

On the other hand, there is some risk in using a real name that we don't control (namely that the tests could become broken and reach out to that real name, which could be controlled by a malicious actor).

I'll take a look at the names we own and see if there's something that seems appropriate to use.